TimeServer/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+
+var endpoint = new IPEndPoint(
+    IPAddress.Loopback,
+    14411);
+TcpListener tcpListener = new (endpoint);
+tcpListener.Start();
+
+var tcpClient = tcpListener.AcceptTcpClient();
+new Thread(() => {
+    var stream = tcpClient.GetStream();
+
+    byte[] buffer = new byte[100];
```

Comment:
no need to initialize the array with a new instance here, if you override it in the next line, anyways. Just du `var buffer = Encoding.ASCII...`